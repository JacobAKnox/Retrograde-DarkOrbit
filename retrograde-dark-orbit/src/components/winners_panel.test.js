import { render } from '@testing-library/react'
import "setimmediate";
import GameOverPanel from './game_over_panel';

test('renders on-screen', () => {
  const page = render(<GameOverPanel/>);
  expect(page).toMatchSnapshot();
});
