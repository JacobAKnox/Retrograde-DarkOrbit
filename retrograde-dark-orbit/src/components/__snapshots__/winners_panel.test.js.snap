// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders on-screen 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="border-4 w-full md:w-1/2 border-blue-400 min-h-[600px] bg-gray-800 md:mx-20 p-4 rounded-xl flex flex-col items-end flex-wrap"
      >
        <div
          class="flex flex-row w-full"
        >
          <div
            class="flex flex-grow m-2"
          >
            <div
              class="text-center bg-slate-900 p-3 rounded-xl w-full m-1"
            >
              <h1
                class="text-2xl text-red-300"
              >
                <b>
                  Team
                   Wins!
                </b>
              </h1>
              <br />
              <div
                class="text-xl text-red-100"
              >
                Players:
                <br />
                <div
                  class="grid grid-cols-5 text-lg text-white"
                >
                  <div
                    class="p-2"
                  >
                     
                    name1
                     
                  </div>
                  <div
                    class="p-2"
                  >
                     
                    name2
                     
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col w-full md:w-1/3 p-4 bg-gray-800 rounded-xl"
          >
            <div
              class="bg-slate-900 text-xl text-white m-1 py-2 px-10 rounded-xl disabled:bg-slate-950 disabled:text-gray-700"
            >
              <div>
                <br />
                <p />
              </div>
            </div>
          </div>
        </div>
        <div
          class="w-full"
        >
          <div
            class="min-h-[200px] bg-gray-800 p-4 rounded-xl flex-col"
          >
            <div
              class="bg-gray-700 w-full h-[200px] p-2 rounded-xl overflow-y-auto"
              id="message-container"
            />
            <div
              class="bg-inherit items-center justify-start w-full h-10 mt-4 flex"
            >
              <input
                aria-label="message"
                class="bg-slate-700 text-white w-3/4 h-10 rounded-xl p-2 hover:bg-slate-600"
                placeholder="message"
                value=""
              />
              <button
                class="bg-slate-700 text-white w-1/4 h-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-950 disabled:text-gray-700"
              >
                Send
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="border-4 w-full md:w-1/2 border-blue-400 min-h-[600px] bg-gray-800 md:mx-20 p-4 rounded-xl flex flex-col items-end flex-wrap"
    >
      <div
        class="flex flex-row w-full"
      >
        <div
          class="flex flex-grow m-2"
        >
          <div
            class="text-center bg-slate-900 p-3 rounded-xl w-full m-1"
          >
            <h1
              class="text-2xl text-red-300"
            >
              <b>
                Team
                 Wins!
              </b>
            </h1>
            <br />
            <div
              class="text-xl text-red-100"
            >
              Players:
              <br />
              <div
                class="grid grid-cols-5 text-lg text-white"
              >
                <div
                  class="p-2"
                >
                   
                  name1
                   
                </div>
                <div
                  class="p-2"
                >
                   
                  name2
                   
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full md:w-1/3 p-4 bg-gray-800 rounded-xl"
        >
          <div
            class="bg-slate-900 text-xl text-white m-1 py-2 px-10 rounded-xl disabled:bg-slate-950 disabled:text-gray-700"
          >
            <div>
              <br />
              <p />
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-full"
      >
        <div
          class="min-h-[200px] bg-gray-800 p-4 rounded-xl flex-col"
        >
          <div
            class="bg-gray-700 w-full h-[200px] p-2 rounded-xl overflow-y-auto"
            id="message-container"
          />
          <div
            class="bg-inherit items-center justify-start w-full h-10 mt-4 flex"
          >
            <input
              aria-label="message"
              class="bg-slate-700 text-white w-3/4 h-10 rounded-xl p-2 hover:bg-slate-600"
              placeholder="message"
              value=""
            />
            <button
              class="bg-slate-700 text-white w-1/4 h-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-950 disabled:text-gray-700"
            >
              Send
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
